src/js/controller/setup-steps.js

```diff
@@ -24,11 +24,21 @@ define([
                 method: _loadBase64Polyfill,
                 depends: []
             },
+            LOAD_VTTCUE_POLYFILL : {
+                method: _loadVTTCuePolyfill,
+                depends: []
+            },
+            LOAD_VTTREGION_POLYFILL : {
+                method: _loadVTTRegionPolyfill,
+                depends: []
+            },
             LOADED_POLYFILLS : {
                 method: _loadedPolyfills,
                 depends: [
                     'LOAD_PROMISE_POLYFILL',
-                    'LOAD_BASE64_POLYFILL'
+                    'LOAD_BASE64_POLYFILL',
+                    'LOAD_VTTCUE_POLYFILL',
+                    'LOAD_VTTREGION_POLYFILL'
```

Comment:
Ideally we would not load these polyfills until we know we need them. Can this be deferred until captions are detected rather than being done at setup?